### ISSUE TITLE
StackedBarChart aria-labelledby attribute value invalid

### ISSUE THREAD

**Issue by beeisabelm** [microsoft/fluentui#13468]
### Environment Information

- **Package version(s)**: charting v2.2.14
- **Browser and OS versions**: All browsers, Windows 10

### Describe the issue:

Running Accessibility Insight for Web, we are seeing the violation of invalid ARIA attribute value on StackedBarChart.

```
Title: WCAG 4.1.1,WCAG 4.1.2: Ensures all ARIA attributes have valid values (.opacityChangeOnHover-223[aria-labelledby="callout23"][focusable="true"]:nth-child(1))
Tags: Accessibility, WCAG 4.1.1, WCAG 4.1.2, aria-valid-attr-value

Issue: Ensures all ARIA attributes have valid values (aria-valid-attr-value - https://dequeuniversity.com/rules/axe/3.5/aria-valid-attr-value?application=msftAI)

Target application: M365 Admin Experiment Controls Examples

Element path: .opacityChangeOnHover-223[aria-labelledby="callout23"][focusable="true"]:nth-child(1)

Snippet: <g class="opacityChangeOnHover-223" data-is-focusable="true" focusable="true" aria-labelledby="callout23" pointer-events="all" tabindex="0"><rect x="0%" y="0" width="66.66666666666666%" height="8" fill="#0078D4"></rect></g>

How to fix: 
Fix all of the following:
  Invalid ARIA attribute value: aria-labelledby="callout23"

Environment: Chrome version 83.0.4103.61

This accessibility issue was found using Accessibility Insights for Web 2.18.2 (axe-core 3.5.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
```
Please refer to `StackedBarChart.base.tsx`
The issue seems to be caused by the `aria-labelledby `on line 280 did not take into account that `Callout `element is null when `!hideTooltip && isCalloutVisible === false`

#### Actual behavior:
**callout23 does not exist on page.**
`<g class="opacityChangeOnHover-223" data-is-focusable="true" focusable="true" aria-labelledby="callout23" pointer-events="all" tabindex="0"><rect x="0%" y="0" width="66.66666666666666%" height="8" fill="#0078D4"></rect></g>`

#### Expected behavior:

use valid ARIA attribute value

### Documentation describing expected behavior

Switch to use `aria-label` with` this.state.selectedLegendTitle + this.state.dataForHoverCard` since this is basically what a Callout element holds in its content.

**Comment by paulgildea** [comment]
@Raghurk is there someone on your team that can look into this issue?

**Comment by Raghurk** [comment]
This is fixed please pull the latest charting library
